SU-SWS/saa_victoria_subtheme
# ADAPT-000: Added smaller max-width for testimonial banner on sidebar …

…nav page.

# READY FOR REVIEW

# Summary
- Fixed padding on sides of testimonial banner when on sidebar nav pages

# Review By (Date)
- 7/22

# Urgency
- medium, was noticed

# Steps to Test

1. pull down the branch
1. clear the cache
2. check a testimonial in-page banner on a page with a side nav, and make sure padding looks good at all sizes, with some nice spacing on the sides.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket - none
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
